docker/supabase/volumes/db/init/04-telescope-schema.sql

```diff
@@ -0,0 +1,41 @@
+-- Case-insensitive text type
+CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA extensions;
+
+-- Registered Telescope users
+CREATE TABLE IF NOT EXISTS telescope_profiles (
+  id text PRIMARY KEY,
+  registered_at timestamptz DEFAULT timezone('utc'::text, now()) NOT NULL,
+  updated_at timestamptz DEFAULT timezone('utc'::text, now()) NOT NULL,
+  display_name citext UNIQUE NOT NULL,
+  first_name text NOT NULL,
+  last_name text NOT NULL,
+  email citext UNIQUE NOT NULL,
+  github_username citext UNIQUE NOT NULL,
+  github_avatar_url text NOT NULL
+);
+
+-- Planet CDOT Feed List, see https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List
+CREATE TABLE IF NOT EXISTS feeds (
+  url text PRIMARY KEY,
+  user_id text references telescope_profiles ON DELETE CASCADE, -- optional, a user can claim an existing feed when they register
```

Comment:
Can `telescope_profiles` be referenced like this or does it need a column too?

```
user_id text references telescope_profiles(id) ON DELETE CASCADE, 
```

vs

```
user_id text references telescope_profiles ON DELETE CASCADE, 
```

Reply:
They are equivalent.
[Table constraints](https://www.postgresql.org/docs/9.2/ddl-constraints.html)

>  in absence of a column list the primary key of the referenced table is used as the referenced column(s).

Reply:
Wow Supabase is really sophisticated

Reply:
It's actually a Postgres Rule 😜

Reply:
I stand corrected XD

Reply:
Feel free to have any question, I learned answering questions too.